infrastructure/persistence/database/reader_simulation_repository.py

```
@@ -0,0 +1,120 @@
+
+    def __init__(self, db=None):
+        self._db = db
+
+    def _get_db(self):
+        if self._db:
+            return self._db
+        from infrastructure.persistence.database.connection import get_database
+        return get_database()
+
+    def ensure_table(self) -> None:
+        """确保表存在（首次调用时自动建表）。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)